### ISSUE TITLE
DON'T DOWNLOAD

### ISSUE THREAD

**Issue by Cucuzacu** [mtgsoftworks/exe2html#1]
The html file downloads a really suspicious looking file called evil.exe. I'm sure it is a virus! Don't download!!!!!!!!

**Comment by mtgsoftworks** [comment]
evil.exe is definitely not a virus. The desired exe file is converted to base64 format, which is embedded in a small html file. Thanks to the algorithm written in HTML, the base64 code is turned into an exe and saved. In the last step, the latest exe file is triggered.

**Comment by Cucuzacu** [comment]
ohhhh ok

**Comment by CoderKing06** [comment]
No, it's very obviously evil. It says so in the name. (This is a joke)

**Comment by willamafton751** [comment]
windows says i cant open evil.exe so idk what to do now

**Comment by hihihihihihi18** [comment]
same what do i do
